yarn.lock

```diff
@@ -3821,6 +3821,11 @@
   resolved "https://registry.yarnpkg.com/@patternfly/patternfly/-/patternfly-2.60.0.tgz#fb319c92514706220007437d7c6e11e9383343f8"
   integrity sha512-oGivL/sudNzQ8ERIfghT1MApQumnixdii/pkHh69fRjyjgo50Np+Ne+j3WAO4NtIqBB5nuSvlI5UnBZegYBB0A==
 
+"@patternfly/patternfly@2.58.0":
+  version "2.58.0"
+  resolved "https://registry.yarnpkg.com/@patternfly/patternfly/-/patternfly-2.58.0.tgz#699f811e12a2372af4b930800826b16b81a280f6"
+  integrity sha512-evVE9/5BJWznY8yXYk0mxq9Cws84GFNLPF9OIaHuOSDIcy8HTQ30bTsnLqrTJLrJ5+o+duYttaNO/buskGJNDw==
+
```

Comment:
Everything else looks ok, but we don't want to update the lock file. That is, we rely on the build automation to bump the patternfly version for all packages.